crates/bazed-core/src/buffer.rs

```diff
@@ -1,3 +1,14 @@
+//! Buffer manages all editing operations on the text structure,
+//! including adjusting carets and selections when edits happen and keeping an undo-history.
+//!
+//! # Some terminology
+//! - **character index**: Index of a character in the text.
```

Comment:
Generally it's best to avoid `character` when defining terminology that involves unicode, code point would be better.

Reply:
Fixed by actually just removing mentions of character completely, character index is not really unsed anywhere else, so that definition was basically useless